### PR TITLE
Added GuardianLines

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Brings together the various places where the same css is used to render the 'Guradian Lines'.

```
type Props = {
    pillar?: Pillar; // For colour
    squiggly?: boolean;
    count?: 4 | 8;
};
```

## Why?
For code reuse

## Link to supporting Trello card
https://trello.com/c/rld7oisF/820-guardianlines

### CODE REVIEW

File: src/web/components/MostViewed/MostViewed.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ function buildSectionUrl(sectionName?: string) {
     const sectionsWithoutPopular = ['info', 'global'];
     const hasSection =
         sectionName && !sectionsWithoutPopular.includes(sectionName);
-    const endpoint = `/c${hasSection ? `/${sectionName}` : ''}.json`;
+    const endpoint = `/most-read${hasSection ? `/${sectionName}` : ''}.json`;
```
Comment (philmcmahon) [inline src/web/components/MostViewed/MostViewed.tsx]: is this a related change?

Comment (oliverlloyd) [reply src/web/components/MostViewed/MostViewed.tsx]: It's fixing a typo from another change but because this PR was built off the Storybook PR, and because it fails silently, I decided to fix it in both places
